src/pages-and-resources/discussions/app-list/AppList.jsx

```diff
@@ -1,35 +1,39 @@
-import React, { useCallback, useEffect } from 'react';
-import PropTypes from 'prop-types';
+import React, { useCallback } from 'react';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { CardGrid, Container } from '@edx/paragon';
 import { useDispatch, useSelector } from 'react-redux';
 
 import { useModels } from '../../../generic/model-store';
+import { selectApp, LOADED, LOADING } from '../data/slice';
 
 import AppCard from './AppCard';
 import messages from './messages';
 import FeaturesTable from './FeaturesTable';
 import AppListNextButton from './AppListNextButton';
+import Loading from '../../../generic/Loading';
 
-import { fetchApps } from './data/thunks';
-import { selectApp, LOADED } from './data/slice';
-
-function AppList({ courseId, intl }) {
+function AppList({ intl }) {
   const dispatch = useDispatch();
-  useEffect(() => {
-    dispatch(fetchApps(courseId));
-  }, [courseId]);
 
   const {
-    appIds, featureIds, status, selectedAppId,
-  } = useSelector(state => state.discussions.appList);
+    appIds, featureIds, status, activeAppId, selectedAppId,
+  } = useSelector(state => state.discussions);
   const apps = useModels('apps', appIds);
   const features = useModels('features', featureIds);
 
   const handleSelectApp = useCallback((appId) => {
     dispatch(selectApp({ appId }));
   }, [selectedAppId]);
 
+  // If selectedAppId is not set, use activeAppId
+  const finalSelectedAppId = selectedAppId || activeAppId;
```

Comment:
`selectedAppId` is not set on load by default.  This uses the value from `activeAppId` instead if that's the case.